src/uss/ZoweUSSNode.ts

```diff
@@ -108,13 +108,13 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
     }
 
     /**
-     * Implements access to profile name
+     * Implements access tto profile name
      * for {IZoweUSSTreeNode}.
      *
      * @returns {string}
      */
     public getProfileName(): string {
-        return this.mProfileName;
+        return this.returnmProfileName();
```

Comment:
This was to fix a Sonar error...Sonar didn't like that getProfileName was returning variable mProfileName instead of profileName (getters should return the variable they are named after) and we couldn't easily rename either profileName or mProfileName, so this was the solution.

I will leave it as-is in case Sonar comes back...